frontend/frontend/private.py

```diff
@@ -13,13 +13,16 @@
 from functools import lru_cache
 import traceback
 from textwrap import dedent
+import asyncio
+from time import sleep
```

Comment:
Why are we sleeping?

Reply:
In the loop to wait for algorithms to reset. The unnecessary import of `sleep` has been removed from `backend/algs/utils.py`.